migration_handler_test.go

```diff
@@ -0,0 +1,305 @@
+package fosite_test
+
+import (
+	//"fmt"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/ory/fosite"
```

Comment:
is that "." intentional?

Reply:
Yup, in Go that means to import that package and treat it as the current package.